cmake/GzRelocatableBinaries.cmake

```diff
@@ -74,19 +74,12 @@ macro(gz_add_get_install_prefix_impl)
       "gz_add_get_install_prefix_impl: missing parameter OVERRIDE_INSTALL_PREFIX_ENV_VARIABLE")
   endif()
 
-
   if(NOT TARGET ${PROJECT_LIBRARY_TARGET_NAME})
     message(FATAL_ERROR
       "Target ${PROJECT_LIBRARY_TARGET_NAME} required by gz_add_get_install_prefix_impl\n"
       "does not exist.")
   endif()
 
-  if(NOT TARGET ${DL_TARGET})
```

Comment:
Since there's another code block that's conditioned on `GZ_ENABLE_RELOCATABLE_INSTALL` on line 101(before PR),  should this stay here with an additional condition on `GZ_ENABLE_RELOCATABLE_INSTALL`?

Reply:
We don't define GZ_ENABLE_RELOCATABLE_INSTALL until after where the code was before the PR.  I moved it to the chunk in line 101.